pkg/logql/log/fmt.go

```diff
@@ -299,14 +300,17 @@ func NewLabelsFormatter(fmts []LabelFmt) (*LabelsFormatter, error) {
 	return lf, nil
 }
 
-func validate(fmts []LabelFmt) error {
+func validate(fmts []LabelFmt, formatRenameErrors bool) error {
 	// it would be too confusing to rename and change the same label value.
 	// To avoid confusion we allow to have a label name only once per stage.
 	uniqueLabelName := map[string]struct{}{}
 	for _, f := range fmts {
-		if f.Name == logqlmodel.ErrorLabel {
+		if f.Name == logqlmodel.ErrorLabel && !formatRenameErrors {
```

Comment:
@cyriltovena Any idea why we prevented renaming the error label previously? I'm inclined to not treat it in a special way.